test/conformance/serial/ptp.go

```
@@ -3418,8 +3803,175 @@ func checkStabilityOfWPCGMUsingMetrics(fullConfig testconfig.TestConfig) {
 	checkPTPNMEAStatus(fullConfig, "1")
 }
 
-func testCaseEnabled(testCase TestCase) bool {
+// checkStabilityOfSimGMUsingMetrics checks simulated T-GM stability
+// using only the processes and metrics available in the simulated environment.
+func checkStabilityOfSimGMUsingMetrics(fullConfig testconfig.TestConfig) {
```

golangci-lint (2.12.2)

[error] func checkStabilityOfSimGMUsingMetrics is unused

(unused)